load_data.py

```diff
@@ -15,13 +15,21 @@
 from data.parse_schema import parse_schema
 from parse_schema import TopicsSchemaParser
 from thresher.models import Article, AnalysisType, TUA, Topic
-
+import pprint;
```

Comment:
Not a big deal but you don't really need ";" in python. It won't break though. Also, I am not seeing pprint added in any of the commits, is it needed?